app/models/concerns/waste_exemptions_engine/can_activate_exemption.rb

```diff
@@ -22,7 +22,15 @@ module CanActivateExemption
       # Transition effects
       def activate_exemption
         self.registered_on = Date.today
-        self.expires_on = Date.today + (WasteExemptionsEngine.configuration.years_before_expiry.years - 1.day)
+
+        self.expires_on =
+          if transient_registration.is_a? WasteExemptionsEngine::RenewingRegistration
+            transient_registration.registration.expires_on +
+              WasteExemptionsEngine.configuration.years_before_expiry.years
```

Comment:
Hard to get my head round this but should this be 3 years minus one day like in line 31? so it expires on the last day

Reply:
a/c says: "the new registration will have an expiry date, three years after the current registrations expiry date"